packages/components/listbox/src/listbox.ts

```diff
@@ -1,23 +1,128 @@
-import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+import { LitVirtualizer } from '@lit-labs/virtualizer';
+import { type ScopedElementsMap, ScopedElementsMixin } from '@open-wc/scoped-elements/lit-element.js';
+import { type PathKeys, getValueByPath } from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type PropertyValues, type TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
 import styles from './listbox.scss.js';
+import { Option } from './option.js';
 
 declare global {
   interface HTMLElementTagNameMap {
     'sl-listbox': Listbox;
   }
+
+  interface ShadowRoot {
+    // Workaround for missing type in @open-wc/scoped-elements
+    createElement<K extends keyof HTMLElementTagNameMap>(
+      tagName: K,
+      options?: ElementCreationOptions
+    ): HTMLElementTagNameMap[K];
+  }
 }
 
-export class Listbox extends LitElement {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type ListboxRenderer<T = any> = (option: T, index: number) => Element | TemplateResult;
```

Comment:
Can we please use `unknown` instead of `any` here?

Reply:
Same